app/Http/Controllers/SupplyController.php

```diff
@@ -242,9 +243,11 @@ public function logNetworkView(Request $request, Adselect $adselect, $id)
         $url = Utils::addUrlParameter($url, 'k', Utils::urlSafeBase64Encode(json_encode($log->our_userdata)));
         $url = Utils::addUrlParameter($url, 'r', Utils::urlSafeBase64Encode($backUrl));
 
-        $response = new RedirectResponse($url);
+        return new Response(null, Response::HTTP_NO_CONTENT);
 
-        return $response;
+//        $response = new RedirectResponse($url);
+//
+//        return $response;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS125)